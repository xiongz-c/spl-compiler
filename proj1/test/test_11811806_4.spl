struct foo1 {
    int attr = ~;
} s1, s2[3]

int f002() {
    return 0;
}

int test_() {
    int a = f002);
    return 0;
}