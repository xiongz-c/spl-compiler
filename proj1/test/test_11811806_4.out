Error type A at Line 2: unknown lexeme ~
Error type B at Line 3: Missing semicolon';'
Error type B at Line 10: Missing semicolon ';'
